### PR TITLE
BZ2012712 OLM 4.9+ InstallPlan keeps installing after error

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.9+

Issue: [BZ2012712](https://bugzilla.redhat.com/show_bug.cgi?id=2012712)

Link to docs preview:

* [Operator groups > Troubleshooting > Membership](https://53573--docspreview.netlify.app/openshift-enterprise/latest/operators/understanding/olm/olm-understanding-operatorgroups.html#olm-operatorgroups-troubleshooting_olm-understanding-operatorgroups)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by michaelryanpeter** [comment]
@jianzhangbjz Would you PTAL and verify if the changes are accurate for 4.9+?

**Comment by kuiwang02** [comment]
> @jianzhangbjz Would you PTAL and verify if the changes are accurate for 4.9+?

yes.

File: modules/olm-operatorgroups-troubleshooting.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@
 ** An incorrect or non-existent service account name is specified in the Operator group.
 --
 +
-If an install plan encounters an invalid Operator group, the CSV is not generated and the `InstallPlan` resource fails with a relevant message. For example, the following message is provided if more than one Operator group exists in the same namespace:
+If an install plan encounters an invalid Operator group, the CSV is not generated and the `InstallPlan` resource keeps installing with a relevant message. For example, the following message is provided if more than one Operator group exists in the same namespace:
```
Comment (tmalove) [inline modules/olm-operatorgroups-troubleshooting.adoc:20]:
```suggestion
If an install plan encounters an invalid Operator group, the CSV is not generated and the `InstallPlan` resource continues to install with a relevant message. For example, the following message is provided if more than one Operator group exists in the same namespace:
```
What you have is fine; this is just another option.

**Comment by michaelryanpeter** [comment]
Merge review squad, would you PTAL?

I can merge and cherrry pick the PR after it has been reviewed.

/label merge-review-needed

**Pull request merged**